backend/app/cli.py

```diff
@@ -87,6 +94,24 @@ def seed(with_model: bool, model_name: str) -> None:
             exclusive_labels=True,
         )
         pipeline = PipelineDB()
+
+        # Create default disconnected source and sink
+        disconnected_source = SourceDB(
+            id="00000000-0000-0000-0000-000000000000",
+            name=DisconnectedSourceConfig().name,
+            source_type=DisconnectedSourceConfig().source_type,
+            config_data={},
+        )
```

Comment:
No need to create instances multiple times, same for sinks:

```suggestion
        disconnected_source_cfg = DisconnectedSourceConfig()
        disconnected_source = SourceDB(
            id="00000000-0000-0000-0000-000000000000",
            name=disconnected_source_cfg.name,
            source_type=disconnected_source_cfg.source_type,
            config_data={},
        )
```